selftest/selftest

```diff
@@ -0,0 +1,27 @@
+#! /bin/bash
+
+set -e
+
+if [ -x "$(command -v fgconsole)" ]
+then
+    trap "sudo chvt $(sudo fgconsole)" EXIT
+fi
+
+if [ -v DISPLAY ]
+then
+    snap run mir-test-tools.gtk3-test
+    snap run mir-test-tools.qt-test
+    snap run mir-test-tools.sdl2-test
+    snap run mir-test-tools.smoke-test
+    snap run mir-test-tools.performance-test
+    snap run mir-test-tools.mir-flutter-app
```

Comment:
There's no such app

Reply:
Whoops! Meant `mir-test-tools.flutter-app`

Reply:
And... that just hangs with "Watches established."

Reply:
.. I don't think that "app" is a test

Reply:
....

Reply:
You are right!

Reply:
Fixed :+1: